src/linker/Linker.Dataflow/ValueNode.cs

```diff
@@ -0,0 +1,1258 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+using TYPE = Mono.Cecil.TypeDefinition;
```

Comment:
Please don't

Reply:
I added this so that we can keep track of how much this file depends on Cecil (if someone ever adds `using Mono.Cecil` at the top, we would see right away).

It's connected to "ValueNode.cs has barely any references to Cecil - the hope is that we would be able to reuse this file for e.g. a Roslyn analyzer" in the pull request description.

If we don't care about reusing this file, I can inline this.

Reply:
I think it's OK to not do this for now - if we get to analyzer and we think it's worth reusing, then we can reintroduce this quite easily.